data/json/mapgen/parks_recreation/cemetary.json

```diff
@@ -0,0 +1,91 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "cemetary" ],
+    "weight": 100,
+    "object": {
+      "rows": [
+        "ssssssssssssssssssssssss",
+        "ssssssssssssssssssssssss",
+        "..........p_p_p.........",
+        "dS.1......_____d...4d.S.",
+        "...dSd4..._____..S...7.d",
+        "..dddd...._____..d......",
+        ".........&_____....ddd..",
+        "...U...d.B_____..UdddU.u",
+        ".b.R.R.R.B_____..Rdbdb.R",
+        ".........._____d........",
+        ".u.u.U...._____....udu..",
+        ".RucuR.R.._____..R.b.R.R",
+        "...u...d.._____.........",
+        ".U........_____....U....",
+        ".Rdb.R.R.._____..R.edb.R",
+        "..dd......_____.....dd..",
+        "...U...u.._____1.....U..",
+        ".R.R.R.b.._____&.b.R.R.R",
+        ".........._____B........",
+        "d..u.u.U.._____B...Udddu",
+        ".R.bdR.R.._____..R.RaR.R",
+        "...ddd....f___f.a.......",
+        "...d........f........d..",
+        ".........d.............."
+      ],
+      "terrain": {
+        "&": "t_grass",
+        ".": "t_grass",
+        "1": "t_tree_young",
+        "4": "t_tree_pine",
+        "7": "t_tree_hickory",
+        "B": "t_grass",
+        "R": "t_grass",
+        "S": "t_shrub",
+        "U": "t_underbrush_harvested_autumn",
+        "_": "t_pavement",
+        "a": "t_shrub_blueberry_harvested",
+        "b": "t_dirt",
+        "c": "t_dirt",
+        "d": "t_dirt",
+        "e": "t_grass",
+        "f": "t_fence_post",
+        "p": "t_pavement_y",
+        "s": "t_sidewalk",
+        "u": "t_underbrush_harvested_winter"
+      },
+      "furniture": {
+        "&": "f_trashcan",
+        ".": "f_null",
+        "1": "f_null",
+        "4": "f_null",
+        "7": "f_null",
+        "B": "f_bench",
+        "R": "f_grave_head",
+        "S": "f_null",
+        "U": "f_null",
+        "_": "f_null",
+        "a": "f_null",
+        "b": "f_grave_head",
+        "c": "f_statue",
+        "d": "f_null",
+        "e": "f_statue",
+        "f": "f_null",
+        "p": "f_null",
+        "s": "f_null",
+        "u": "f_null"
```

Comment:
Pretty sure nulls don't need to be (and thus shouldn't) be explicitly declared.
The exception is when using palettes and overriding an existing piece.